security/security-design/fn-datasafe-dbaudit-to-oci-logging/terraform/main.tf

```diff
@@ -0,0 +1,31 @@
+###############################################################################
+# Copyright (c) 2022, 2023, Oracle and/or its affiliates.  All rights reserved.
+# This software is dual-licensed to you under the Universal Permissive License
+# (UPL) 1.0 as shown at https://oss.oracle.com/licenses/upl.
+###############################################################################
+#
+# Author: Fabrizio Zarri
+#
+################################################################################
+
+
+
+module "setup-network" {
+  source = "./modules/network"
+  count  = var.create_network ? 1 : 0
+  compartment_ocid = var.compartment_ocid
+  VCN-CIDR = var.VCN-CIDR
```

Comment:
Terraform naming convention is all lower case and words separated by '_'.

Reply:
I have update the name of all variables and references. Now are only present lower case and words separated by '_'.